### PR TITLE
Kernels folder and example

### PR DESCRIPTION
**Pull request by Jacfomg**
Add Kernel folder path and file for q0. Should work with https://github.com/qiboteam/qibolab/pull/629.

### CODE REVIEW

**Comment by Jacfomg** [comment]
Tests are failing due to qblox @PiergiorgioButtarini any idea why ?

**Comment by AleCandido** [comment]
> Tests are failing due to qblox @PiergiorgioButtarini any idea why ?

Now they are working. After the release, even *stable* will work.

File: iqm5q.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 TWPA_ADDRESS = "192.168.0.35"
 
 N_QUBITS = 5
+KERNEL_FOLDER = "qibolab_platforms_qrc/iqm5q_kernels/"
```
Comment (andrea-pasquale) [inline iqm5q.py:24]:
```suggestion
KERNEL_FOLDER =pathlib.Path(__file__).parent/ "iqm5q_kernels"
```
I think in this way it should be better.

File: iqm5q.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 )
 
 RUNCARD = pathlib.Path(__file__).parent / "iqm5q.yml"
+KERNEL_FOLDER = pathlib.Path(__file__).parent / "iqm5q_kernels/"
```
Comment (AleCandido) [inline iqm5q.py:20]: Let's just use a single folder for all extra stuffs
```suggestion
FOLDER = pathlib.Path(__file__).parent / "iqm5q"
```

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Review by stavros11 — APPROVED** [review]
We should merge the qibolab PR first and make sure that the workflow using main passes.

File: iqm5q.yml [hunk 3]
```diff
@@ -228,8 +228,9 @@ characterization:
             state1_voltage: 0
             mean_gnd_states: [-0.0051689363924847615, 0.06828212233945871]
             mean_exc_states: [-0.773457961897901, -0.44969928873718545]
-            threshold: 0.43144500370808087
-            iq_angle: 2.5483919327086055
+            threshold: 0.501546
+            iq_angle: 2.672
+            kernel_path: "kernel_q0"
```
[the remark this answers is not in the document]
Comment (Jacfomg) [reply iqm5q.yml:233]: I think it makes more sense in singular, thanks for spotting it.

**Pull request merged**